.github/workflows/icu_ci.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#
+# GitHub Action configuration script for ICU continuous integration tasks.
+
+name: GHA CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '*'
+
+jobs:
+
+  # ICU4J build and unit test
+  icu4j-build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout and setup
+        uses: actions/checkout@v2
+        with:
+          java-version: 1.11
+          lfs: true
+      - name: Checkout lfs objects
+        run: git lfs pull
+      - name: ICU4J
+        run: |
+          cd icu4j;
+          ant init;
+          ant check;
+
+  # clang release build with some options to enforce useful constraints.
+  # Includes dependency checker on an in-source, optimized build.
+  # Includes checking @draft etc. API tags vs. ifndef guards like
+  # U_HIDE_DRAFT_API and U_FORCE_HIDE_DRAFT_API.
+  # (FORCE guards make this tool pass but won't compile to working code.
+  # See the testtagsguards.sh script for details.)
+  clang-release-build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: ICU4C with clang
+        run: |
+          sudo apt-get -y install doxygen;
+          cd icu4c/source;
+          ./runConfigureICU Linux --enable-layoutex=no;
+          make -j 2;
+          make -j 2 check;
+          # ( cd test/depstest && python3 depstest.py ../../../source/ ) ICU-21436
```

Comment:
Please spend time to figure out why depstest is failing in the GitHub Actions environment.  It was passing in the Travis environment.  I have seen depstest fail in Clang where it doesn't fail in GCC; it may have something to do with the standard library environment.

Reply:
I have an ICU ticket open (ICU-21436) and it can be assigned to me next Wednesday.
This is my last afternoon till Jan. 12 and I think merging this PR even with this ticket open is still beneficial for public ICU so migration away from Travis CI can continue.
Something is brittle, it fails in my Linux machine as well.

Reply:
Looks like bcmp() should be ok.
Would you mind trying to change the deps txt file?

After this line: https://github.com/unicode-org/icu/blob/master/icu4c/source/test/depstest/dependencies.txt#L60
add the following two:
````
    # BSD version of memcmp().
    bcmp
```

If there is no time now, I am ok with merging this PR and improving on it later.

Reply:
Updated dependencies.txt, enabled depstest, squashed and pushed.